cli.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'thor'
+require 'exif'
+
+class CLI < Thor
+  desc "scan", "Recursively reads all of the images from the supplied directory"
+  option :path, type: :string, required: false, desc: "Optional parameter that allows any other directory to be passed in."
+
+  def scan
+    File.open("report.csv", "w") do |report|
+      Dir.glob("**/*").each do |file|
+        next unless File.file?(file)
+
+        data = get_exif(file)
+        next unless data
+
+        report.puts(:"File:#{file}")
+        report.puts(:"GPS data:#{data[:gps]}")
+        report.puts(:"--------------------------")
```

Comment:
Nice! Could you share a bit more the why behind adding `:` to the puts method? Although adding it works, we usually just call puts with the string: `puts "----------------"`

Reply:
To be honest, I didn't even notice it. I think I made a typo in the first line and then just duplicated it. I'm surprised it works correctly. As I understand it, I kind of declared a symbol)))